scapy/contrib/automotive/someip.py

```diff
@@ -131,6 +130,10 @@ class SOMEIP(Packet):
     def get_payload_cls_by_srv_id(pkt, lst, cur, remain):
         return SOMEIP.payload_cls_by_srv_id.get(pkt.srv_id, Raw)
 
+    def default_payload_class(self, payload):
+        # type: (bytes) -> Type[Packet]
```

Comment:
I typically use
`bind_layers(A, A)`
for this, instead.

Reply:
fixed

Reply:
Apparently bind_layers doesn't work. Should I revert?